velox/dwio/parquet/tests/writer/ParquetWriterTest.cpp

```diff
@@ -146,6 +147,110 @@ TEST_F(ParquetWriterTest, compression) {
   assertReadWithReaderAndExpected(schema, *rowReader, data, *leafPool_);
 };
 
+TEST_F(ParquetWriterTest, toggleDataPageVersion) {
+  auto schema = ROW({"c0"}, {INTEGER()});
+  const int64_t kRows = 1;
+  const auto data = makeRowVector({
+      makeFlatVector<int32_t>(kRows, [](auto row) { return 987; }),
+  });
+
+  // Write Parquet test data, then read and return the DataPage
+  // (thrift::PageType::type) used.
+  const auto testDataPageVersion =
+      [&](std::optional<std::unordered_map<std::string, std::string>>
```

Comment:
nit: You can make these `std::unordered_map<std::string, std::string>` and then simply pass an empty map `{}` where needed.
Then you don't have to check when creating the connectorConfig.

Reply:
That is true - I just felt it may be more readable/clear to explicitly denote them as optionals, but it is needlessly clunky. Reverted them to concrete values.